interfaces/Portalicious/src/app/pages/project-payments/components/payment-summary-card/payment-summary-card.component.html

```diff
@@ -0,0 +1,80 @@
+<p-card
+  styleClass="[&_.p-card-content]:flex-end group relative h-full border border-transparent *:h-full focus-within:border-purple-500 hover:border-purple-500 [&_.p-card-title]:flex-grow"
+>
+  <ng-template pTemplate="title">
+    <a
+      [routerLink]=""
+      class="after:absolute after:inset-0 after:block group-focus-within:text-purple group-hover:text-purple"
+    >
```

Comment:
FYI I added a comment to your refactor task based on the fact that we are also doing quite some copy-pasting here too:
![image](https://github.com/user-attachments/assets/bcb164e7-b000-4954-a8c1-241b8385eaf8)